content/metadata.yaml

```diff
@@ -17,6 +17,15 @@ author_info:
     affiliations:
       - Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania
     funders: GBMF4552
+  -
+    github: slochower
+    name: David R. Slochower
+    initials: DRS
+    orcid: 0000-0003-3928-5050
+    twitter: drslochower
+    email: slochower@gmail.com
+    affilitations:
+      - Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego, 9500 Gilman Drive, La Jolla, CA 92093
```

Comment:
I was wondering about that... Usually the address is part of the affiliation when I submit my information to journals. I've got no problem removing it now, but should there be an entry for that in the metadata so that it can be included (or not) depending on the template?

Reply:
Also along those lines, what about a flag for corresponding author status? The default could be no, but if one author has in their `yaml` block `corresponding: true` then the template could detect it and add an asterisk (or whatever)? Same idea could be used for `contributed equally: true` or something similar? (Obviously these topics could be migrated to an issue or PR on `manubot` if useful.)

Reply:
Also along those lines, what about a flag for corresponding author status? The default could be no, but if one author has in their `yaml` block `corresponding: true` then the template could detect it and add an asterisk (or whatever)? Same idea could be used for `contributed equally: true` or something similar? (Obviously these topics could be migrated to an issue or PR on `manubot` if useful.)

Reply:
Let's create an issue in manubot-rootstock if we want to modify metadata.yaml.  We should do it upstream so that all derivative manuscripts could take advantage of the updates.

Briefly, an address line does make sense to me.  At one point in deep review https://github.com/greenelab/deep-review/pull/550 I added explicit support for corresponding authors.  We then discussed supporting arbitrary symbols for co-first, corresponding, or other annotations in https://github.com/greenelab/manubot-rootstock/issues/7 and https://github.com/greenelab/manubot-rootstock/pull/10.  That's what we did for [deep review](https://github.com/greenelab/deep-review/blob/5df298135b65c72d164f69571fb303c5d1db21b2/content/metadata.yaml#L20), but it isn't in manubot-rootstock currently.